### ISSUE TITLE
How to use multiple GPUs for End Tasks?

### ISSUE THREAD

**Issue by KuangHaofei** [danielgordon10/vince#7]
Hi,

Thanks for your nice work. But I have a question about your training setting about multiple GPU training for end tasks.

In your example train end task bash files (such as `train_kinetics_400.sh`), there are two different arguments to assign GPU ids:`pytorch-gpu-ids` and `feature-extractor-gpu-ids`. In `tarin_kinetics_400.sh`, both of them are `0`.

My question is: Can we use multiple GPUs to train the end task? How to modify the two arguments (`pytorch-gpu-ids` and `feature-extractor-gpu-ids`) to achieve it? For example:
- both `pytorch-gpu-ids` and `feature-extractor-gpu-ids` use same GPUs:
    ```shell
    --pytorch-gpu-ids 0,1,2,3 \
    --feature-extractor-gpu-ids 0,1,2,3 \
    ```
- or `pytorch-gpu-ids` and `feature-extractor-gpu-ids` use different GPUs:
    ```shell
    --pytorch-gpu-ids 0,1,2,3 \
    --feature-extractor-gpu-ids 4,5,6,7 \
    ```
- or we can specify any GPUs for them:
    ```shell
    --pytorch-gpu-ids 0,1,2,3,4,5,6,7 \
    --feature-extractor-gpu-ids 0,1,2,3 \
    ```
  
Thank you very much!

**Comment by danielgordon10** [comment]
Multi-GPU support should work for end tasks. https://github.com/danielgordon10/vince/blob/master/models/vince_model.py#L35 shows that the model uses the feature extractor gpu ids. Since the final network for each end task is a single layer, it's not advantageous to put it on Multi-GPUs. So you can set `--feature-extractor-gpu-ids` to whatever you want, but you won't get any advantage out of setting `--pytorch-gpu-ids` to multiple numbers, it will just use the first one.
